scrapers_next/pa/committees.py

```diff
@@ -1,105 +1,603 @@
-from spatula import CSS, HtmlPage, HtmlListPage, SelectorError
+from spatula import HtmlPage, SelectorError, XPath, URL, SkipItem
 from openstates.models import ScrapeCommittee
 
 
```

Comment:
Let's add this docstring here to serve as documentation for those running the scrapers of which classes should be run to get all committees:
```
"""
The classes that should be run to ensure every committee is scraped include:
- House
- Senate
- JointCommitteeList
- LegReapportionmentComm
- LocalGovernmentComm
- LegBudgetFinanceComm
- LegAuditAdvisoryComm
- StateGovernmentComm
- CommissionOnSentencing
- CapitolPreservationComm
"""
```